vue/components/ui/molecules/select-checkboxes/select-checkboxes.vue

```diff
@@ -1,21 +1,18 @@
 <template>
-    <select-ripe
-        class="select-checkboxes"
-        v-bind:options="selectOptions"
-        v-bind:value="'checkbox-group'"
-        v-bind:disabled="disabled"
-        v-bind="{
-            autoScroll: false,
-            maxHeight: 210,
-            ...selectProps
-        }"
-    >
-        <template v-slot:checkbox-group>
-            <div class="checkboxes" v-on:click.stop>
-                <checkbox-group v-bind:items="_items" v-bind:values.sync="valuesData" />
-            </div>
-        </template>
-    </select-ripe>
+    <div v-on:keydown.exact.stop="event => onKey(event.key)">

```

Comment:
This seems a very big change that has everything to break the select-checboxes in other places. I think this can't be done this way.

If the reason you added this div was just to do get the `on:key` event, try to just emit [this event in select-ripe](https://github.com/ripe-tech/ripe-components-vue/blob/a2b9d2aa0e199d59211232f6cf15d950ef052206/vue/components/ui/molecules/select/select.vue#L26),

Reply:
Emitting an event in the select-ripe also worked, I did test that. I see what you mean with that, and I'll do those changes!